### PR TITLE
docs(aws): add docs

### PR DESCRIPTION
**Pull request by georgettica**
- move pagerduty docs to it's own folder

### CODE REVIEW

File: pkg/aws/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### extract the external ID
+```
+I= #Incident ID
+pd rest:get -e=/incidents/$I/alerts | jq -r '.alerts[].body.details.notes' | yq .cluster_id
+```
+
+### login to the owner cluster
+using a private script, we can login to the cluster that owns the cluster we want to check (as it holds the aws account)
+-- DM privately to get the details
```
Comment (ramonbutter) [inline pkg/aws/README.md:23]: what does that mean?

Comment (georgettica) [reply pkg/aws/README.md:23]: replied privately

**Pull request merged**